src/components/UIShared/ConversationActionsShortcut.vue

```diff
@@ -0,0 +1,152 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { computed } from 'vue'
+import { isNavigationFailure, NavigationFailureType } from 'vue-router'
+import { useRouter, useRoute } from 'vue-router/composables'
+
+import IconCheckUnderline from 'vue-material-design-icons/CheckUnderline.vue'
+import IconDelete from 'vue-material-design-icons/Delete.vue'
+
+import { showError } from '@nextcloud/dialogs'
+import { t, getCanonicalLocale } from '@nextcloud/l10n'
+
+import NcButton from '@nextcloud/vue/components/NcButton'
+import { spawnDialog } from '@nextcloud/vue/functions/dialog'
+
+import ConfirmDialog from '../../components/UIShared/ConfirmDialog.vue'
+
+import { useStore } from '../../composables/useStore.js'
+import { CONVERSATION } from '../../constants.ts'
+import { hasTalkFeature, getTalkConfig } from '../../services/CapabilitiesManager.ts'
+
+const supportsArchive = hasTalkFeature('local', 'archived-conversations-v2')
+const retentionEventPeriod = getTalkConfig('local', 'conversations', 'retention-event')
+const retentionPhonePeriod = getTalkConfig('local', 'conversations', 'retention-phone')
+
+const props = defineProps<{
+	token: string,
+	objectType: string,
+	isHighlighted: boolean,
+}>()
+
+const store = useStore()
+const router = useRouter()
+const route = useRoute()
+
+const isModerator = computed(() => store.getters.isModerator)
+
+const expirationDuration = computed(() => {
+	if (props.objectType === CONVERSATION.OBJECT_TYPE.EVENT) {
+		return retentionEventPeriod
+	} else if (props.objectType === CONVERSATION.OBJECT_TYPE.PHONE_TEMPORARY) {
+		return retentionPhonePeriod
+	}
+	return 0
+})
+
+const descriptionLabel = computed(() => {
+	const expirationDurationFormatted = new Intl.RelativeTimeFormat(getCanonicalLocale(), { numeric: 'auto' }).format(
```

Comment:
```suggestion
	const expirationDurationFormatted = new Intl.RelativeTimeFormat(getLanguage(), { numeric: 'always' }).format(
```
![image](https://github.com/user-attachments/assets/e1738e0e-da1b-4c39-9c2f-723e0382b5b4)
<img width="345" alt="image" src="https://github.com/user-attachments/assets/c04bddb0-cca1-4df6-ad5f-6006b2a8e528" />